content/ko/examples/pods/pod-nginx-specific-node.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  nodeName: foo-node # schedule pod to specific node
```

@coolguyhong 사소한 의견 드립니다.
코드 내부의 주석의 경우에도 한글화를 진행하고 있습니다.

```suggestion
  nodeName: foo-node # 특정 노드에 파드 스케줄
```

---

반영했습니다.